examples/nvme_linux_test.go

```diff
@@ -3,10 +3,11 @@ package test
 import (
 	"bytes"
 	"fmt"
-	"github.com/anatol/smart.go"
-	"github.com/stretchr/testify/require"
 	"os/exec"
 	"testing"
+
+	"github.com/anatol/smart.go"
```

Comment:
In my other project I was using such grouping. But Intellij IDEA keeps merging it back into one set when I do file modifications. I am tired to fight with Intellij and decided to give a try for "one import group" idea here.

In addition neither `go fmt` nor `revive` seems care about imports grouping. Is there any code style that says third-party imports need to be a separate group?

Reply:
Hmmm.. Using Goland myself, but I set up `goimports`. You should be able to change it.

Reply:
Thanks! I knew there should be a way to do it. I am surprised that I have not found it earlier.